app/src/main/AndroidManifest.xml

```diff
@@ -1,23 +1,25 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest package="io.ooni.mk.nuvolari.nuvolari"
-          xmlns:android="http://schemas.android.com/apk/res/android">
+<manifest
+    xmlns:tools="http://schemas.android.com/tools"
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    package="io.ooni.mk.nuvolari.android">
 
     <uses-permission android:name="android.permission.INTERNET" />
 
     <application
-        android:allowBackup="true"
+        android:allowBackup="false"
```

Comment:
So, this means that the app data will never hit the cloud for backup, correct?

Reply:
Yes, you can read more here: https://developer.android.com/guide/topics/data/backup

For this demo app, not needed at all.

Reply:
Yeah, I agree!